### DISCUSSION TITLE
Does not seem to be working anymore or with Pi Zero W

### DISCUSSION THREAD

**Discussion by ngen33r** [sethvoltz/OctoPrint-DisplayPanel#94] [General]
This project has not seemed to have any updates in a few years. I am trying to make it with with a PI Zero W and I get one blink and then blank screen. The other SSD1306 plugin seems to work fine but has been updated in 2023. Is there a way to debug this or get it going? I really would like to have the IP address and other info on this OLED. Hardware confirmed working well. 

I am not using any cameras and minimal plugins so I am not concerned with speed at the moment. If needed I can get the Zero 2 but again I won't be using a camera.

**Comment by sethvoltz** [comment]
Hey @ngen33r, I don't currently have an active OctoPrint instance running and at the time it fit all the needs and hardware I had on hand. However, I am happy to review PRs if you want to take a crack at putting one together and testing on your hardware.

**Comment by ngen33r** [comment]
I would love to update this and get it working again. Just tell me what 
I need to do.


------ Original Message ------
From "Seth Voltz" ***@***.***>
To "sethvoltz/OctoPrint-DisplayPanel" 
***@***.***>
Cc "ngen33r" ***@***.***>; "Mention" ***@***.***>
Date 10/4/2023 1:33:48 PM
Subject Re: [sethvoltz/OctoPrint-DisplayPanel] Does not seem to be 
working anymore or with Pi Zero W (Discussion #94)

**Comment by ngen33r** [comment]
Is there a log or something I can send to debug the issue?
